scram-tools.file/tools/conifer/conifer.xml

```diff
@@ -0,0 +1,7 @@
+<tool name="conifer" version="@TOOL_VERSION@">
+  <client>
+    <environment name="CONIFER_BASE" default="@TOOL_ROOT@"/>
+    <environment name="INCLUDE" default="$CONIFER_BASE/conifer/backends/cpp/include"/>
```

Comment:
please change `$CONIFER_BASE/conifer/backends/cpp/include`  to `$CONIFER_BASE/include` ( as you copied the header in `%{i}/include` ).

Also please add `<use name="json"/>`